Source/Core/VideoBackends/Vulkan/VKSwapChain.cpp

```diff
@@ -157,27 +157,63 @@ bool SwapChain::SelectSurfaceFormat()
   // If there is a single undefined surface format, the device doesn't care, so we'll just use RGBA
   if (surface_formats[0].format == VK_FORMAT_UNDEFINED)
   {
-    m_surface_format.format = VK_FORMAT_R8G8B8A8_UNORM;
+    m_surface_format.format = VK_FORMAT_A2R10G10B10_UNORM_PACK32;
```

Comment:
Why did this change?

Reply:
(please update the comment if you're going to keep this)

Reply:
The comment on top still stands no? It specified that if the supported format is unknown, any is fine?

Reply:
It's not 'RGBA' anymore is it?

Reply:
Do you know if this is actually a common case?
I should probably revert this unless it's a common case on PCs, which I doubt.

Reply:
Sorry, not sure.

Reply:
Ok I've been reading around about this. It's an exception case from the early days of VK where the function to check the supported surface formats wasn't there (or the result didn't have to be specified).
I've reverted that line, it should safer, and also ended up changing `VK_FORMAT_A2R10G10B10_UNORM_PACK32` for `VK_FORMAT_A2B10G10R10_UNORM_PACK32`, because that's what's actually commonly supported by GPUs.